src/index.html

```diff
@@ -15,8 +15,55 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://127.0.0.1:5500/src/index.html">
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="logo"
```

Comment:
The `alt` attribute should be more descriptive to improve accessibility. Instead of just 'logo', consider something like 'Company Logo' or 'Moyo Logo'.